classes/ConfigurationKPI.php

```diff
@@ -19,152 +19,294 @@
  * needs please refer to http://www.prestashop.com for more information.
  *
  * @author    PrestaShop SA <contact@prestashop.com>
- * @copyright 2007-2015 PrestaShop SA
+ * @copyright 2007-2016 PrestaShop SA
  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+/**
+ * Class ConfigurationKPICore
+ */
 class ConfigurationKPICore extends Configuration
 {
     public static $definition_backup;
 
+    /**
+     * Set KPI definition
+     */
     public static function setKpiDefinition()
     {
         ConfigurationKPI::$definition_backup = Configuration::$definition;
         Configuration::$definition['table'] = 'configuration_kpi';
         Configuration::$definition['primary'] = 'id_configuration_kpi';
     }
 
+    /**
+     * Unset KPI definition
+     */
     public static function unsetKpiDefinition()
     {
         Configuration::$definition = ConfigurationKPI::$definition_backup;
     }
 
-    public static function getIdByName($key, $id_shop_group = null, $id_shop = null)
+    /**
+     * Get ID by name
+     *
+     * @param string   $key         Configuration key
+     * @param int|null $idShopGroup ShopGroup ID
+     * @param int|null $idShop      Shop ID
+     *
+     * @return int ConfigurationKPI ID
+     */
+    public static function getIdByName($key, $idShopGroup = null, $idShop = null)
     {
         ConfigurationKPI::setKpiDefinition();
-        $r = parent::getIdByName($key, $id_shop_group, $id_shop);
+        $configurationKpi = parent::getIdByName($key, $idShopGroup, $idShop);
         ConfigurationKPI::unsetKpiDefinition();
-        return $r;
+
+        return $configurationKpi;
     }
 
+    /**
+     * Load configuration
+     */
     public static function loadConfiguration()
     {
         ConfigurationKPI::setKpiDefinition();
-        $r = parent::loadConfiguration();
+        parent::loadConfiguration();
         ConfigurationKPI::unsetKpiDefinition();
-        return $r;
```

Comment:
@maximebiloe I think for this function, `loadConfiguration` it's not needed because it's return nothing.
Others yes, but he didn't remove the return, so I think that is ok.